redbot/cogs/general/general.py

```diff
@@ -218,35 +218,93 @@ def __init__(self):
-        online = len([m.status for m in guild.members if m.status != discord.Status.offline])
+        online = len([m.status for m in guild.members if m.status == discord.Status.online])
+        idle = len([m.status for m in guild.members if m.status == discord.Status.idle])
+        dnd = len([m.status for m in guild.members if m.status == discord.Status.dnd])
+        offline = len([m.status for m in guild.members if m.status == discord.Status.offline])
         total_users = len(guild.members)
+        bots = len([a for a in ctx.guild.members if a.bot])
+        humans = len([a for a in ctx.guild.members if a.bot == False])
         text_channels = len(guild.text_channels)
         voice_channels = len(guild.voice_channels)
         passed = (ctx.message.created_at - guild.created_at).days
-        created_at = _("Since {date}. That's over {num} days ago!").format(
+        created_at = _("Created on **{date}**. That's over **{num}** days ago !").format(
             date=guild.created_at.strftime("%d %b %Y %H:%M"), num=passed
         )
+        joined_at = guild.me.joined_at
+        since_joined = (ctx.message.created_at - joined_at).days
+        bot_joined = joined_at.strftime("%d %b %Y %H:%M:%S")
+        joined_on = (_(f"{ctx.bot.user.name} joined this server on {bot_joined}. That's over {since_joined} days ago !"))
         data = discord.Embed(description=created_at, colour=(await ctx.embed_colour()))
-        data.add_field(name=_("Region"), value=str(guild.region))
-        data.add_field(name=_("Users"), value=f"{online}/{total_users}")
-        data.add_field(name=_("Text Channels"), value=str(text_channels))
-        data.add_field(name=_("Voice Channels"), value=str(voice_channels))
-        data.add_field(name=_("Roles"), value=str(len(guild.roles)))
-        data.add_field(name=_("Owner"), value=str(guild.owner))
-        data.set_footer(text=_("Server ID: ") + str(guild.id))
-
+        if total_users > 8000:
+            data.add_field(name=_("Members :"),
+            value=_(f"Total users : **{total_users}**\nHumans : **{humans}** • Bots : **{bots}**\n📗 `{online}` 📙 `{idle}`\n📕 `{dnd}` 📓 `{offline}`"))
+        else:
+            data.add_field(name=_("Members :"),
+            value=_(f"Total users : **{total_users}**\nHumans : **{humans}** • Bots : **{bots}**\n📗 `{online}` 📙 `{idle}` 📕 `{dnd}` 📓 `{offline}`"))
+        data.add_field(name=_("Channels :"), value=_(f"💬 Text : **{text_channels}**\n🔊 Voice : **{voice_channels}**"))
+        data.Empty
+        data.add_field(name=_("Utility :"), value=_(f"Owner : **{guild.owner}**\nRegion : **{region[str(guild.region)]}**\nVerif. level : **{verif[int(guild.verification_level)]}**\nServer ID : **{guild.id}**"))
+        data.add_field(name=_("Misc :"), value=_(f"AFK channel : **{guild.afk_channel}**\nAFK Timeout : **{guild.afk_timeout}sec**\nCustom emojis : **{len(guild.emojis)}**\nRoles : **{len(guild.roles)}**"))
+        if guild.features != []:
```

Comment:
```suggestion
        if guild.features:
```

empty lists evaluate as false.